src/main/java/de/doubleslash/keeptime/view/SettingsController.java

```diff
@@ -391,10 +391,13 @@ private void initImportButton() {
             final String url = applicationProperties.getSpringDataSourceUrl();
             final String username = applicationProperties.getSpringDataSourceUserName();
             final String password = applicationProperties.getSpringDataSourcePassword();
-            // TODO: add an option at the next release to make the "FROM_1X flag" configurable. E.g. if we upgrade (in
-            // the release after) from H2 version 2.x to 2.x we must not set the "FROM_1X flag".
-            new RunScript().runTool("-url", url, "-user", username, "-password", password, "-script", file.toString(),
-                  "-options", "FROM_1X");
+
+            if (file.getName().contains("version-1")) {
+               new RunScript().runTool("-url", url, "-user", username, "-password", password, "-script", file.toString(),

```

Comment:
When using filename (which the user may change while exporting) I would check for contains "H2-version-1." (to make the check more obvious what version it checks).
Also in the if branch please add a info log statement to log out that the FROM_1X feature is used.

Reply:
Add a log info statement